repository/src/lib.rs

```diff
@@ -9,3 +15,118 @@ pub type Tag = String;
 pub struct CommitHash {
     pub hash: [u8; 32],
```

Comment:
Isn't this [u8; 20]? CommitHash consists of 40 hex characters

Reply:
It’s 256 bits for the hash value.

Reply:
@JeongHunP You are right. I fixed it.